packages/browser-repl/src/components/ace-autocompleter-adapter.ts

```diff
@@ -0,0 +1,47 @@
+import { Autocompleter, Completion } from '../lib/autocompleter/autocompleter';
```

Comment:
there is an [ace-autocomplete](https://github.com/mongodb-js/ace-autocompleter) which may or may not be helpful here to get your line positions and other ace-y things